README.md

```diff
@@ -16,35 +15,35 @@
   - [Наглядная схема устройства факультетов](#схема-факультетов)
 - [Сокращения, сленг и ссылки](#сокращения-сленг-и-ссылки)
   - [Предметы](#предметы)
-  - [Здания](#здания)
-    - [Корпуса университета](#корпуса-университета)
-    - [Общежития](#общежития)
+  - [Корпуса университета](#корпуса-университета)
+  - [Общежития](#общежития)
   - [Сайты](#сайты)
   - [Прочее](#прочее)
 - [Сборник полезных источников](#сборник-полезных-источников)
   - [Пояснения к иконкам содержимого](#пояснения-к-иконкам-содержимого)
   - [Полезные источники](#полезные-источники)
 - [Гелиус](#гелиус)
   - [Подключение и загрузка файлов](#Подключение)
-    - [Авторизация на se.ifmo.ru](#авторизация)
+    - [Авторизация на se.ifmo.ru](#авторизация-в-seifmoru)
     - [Подключение через консоль (powershell, bash, zsh, e.t.c)](#подключение-через-консоль-powershell-bash-zsh-etc)
     - [Загрузка файлов через консоль (powershell, bash, zsh, e.t.c)](#загрузка-файлов-через-консоль-powershell-bash-zsh-etc)
     - [Создание `.bat` - файла для авто-подключения к серверу (консоль + для загрузки файлов)](#создание-bat---файла-для-авто-подключения-к-серверу-консоль--для-загрузки-файлов)
     - [PuttY (консоль)](#putty-консоль)
     - [WinSCP (для загрузки файлов)](#winscp-для-загрузки-файлов)
     - [FileZilla (для загрузки файлов)](#filezilla-для-загрузки-файлов)
     - [Git (для загрузки файлов)](#git-для-загрузки-файлов)
-  - [Лайфхаки по работе с сервером](#лайфхаки-по-работе-с-сервером)
-    - [Сочетания клавиш и команды терминала](#команды)
-    - [Cоздание перманентной переменной окружения](#создание-перманентной-переменной-окружения)
-    - [Траблы](#траблы)
-    - [Приколы](#Приколы)
+  - [Сочетания клавиш и команды терминала](#cочетания-клавиш-и-команды-терминала)
+  - [Траблы](#траблы)
+    - [JAVA: ошибка переполнения кучи или java по дефолту 11 версии, BindException](#java-ошибка-переполнения-кучи-или-java-по-дефолту-11-версии-bindexception)
+    - [Проблемы с базой данных и клиентскими и серверными приложениями (не запускается, переполнение кучи и т.д.)](#проблемы-с-базой-данных-и-клиентскими-и-серверными-приложениями-не-запускается-переполнение-кучи-и-тд)
+  - [Cоздание перманентной переменной окружения](#создание-перманентной-переменной-окружения)
+  - [Приколы](#Приколы)
 - [Система оценивания БаРС и разбалловка по всем предметам](#система-оценивания-барс-и-разбалловка-по-всем-предметам)
   - [Наглядная диаграмма разбалловки БаРС](#разбалловка)
   - [Подробные гугл журналы для самостоятельного учёта успеваемости по ВСЕМ предметам конкретного семестра ВТ/СУиР](#журналы)
 - [ITMO ENTER](#itmo-enter)
   - [Фотки презентаций, тестов и QR-кодов с ITMO Enter 2022](#itmo-enter-фотки)
-  - [Можно ли получить незачёт?](#itmo-enter-незачёт)
+  - [Можно ли получить незачёт?](#можно-ли-получить-незачёт)
```

Comment:
It's wrong, because this paragraf belongs ONLY to itmo enter. 

But "можно-ли-получить-незачёт" could be ANY subject, not only itmo-enter. To avoid repeating - make it as it was, there is no reasong to rename it